views/007_events.sql

```diff
@@ -218,48 +218,6 @@ WHERE
 GROUP BY
   name;
 
--- Insert team updates in event_queue
-CREATE
-OR REPLACE FUNCTION insert_team_in_event_queue () RETURNS TRIGGER AS $$
```

Comment:
Removed this from being pushed to the event queue because the event consumer in IC were simply clearing the `teams` & `responder` cache. https://github.com/flanksource/incident-commander/blob/1265484533b60bfb1196c017e9769dfc9151a617/events/teams.go#L34-L70

Instead now we have PG Notify on the teams table.